server/src/query/activity.ts

```diff
@@ -172,12 +173,18 @@ export async function deleteContent({
     );
   }
 
+  // Delete descendants
   const ids = await getDescendantIds(contentId);
-
-  return prisma.content.updateMany({
-    where: { id: { in: [contentId, ...ids] } },
-    data: { isDeleted: true },
+  const deleteDescendants = prisma.content.updateMany({
```

Comment:
The descendants and the content itself will have two different deleted on dates. Create the `new Date()` and save it to a variable so they are identical.